charts/rancher-backup/files/default-resourceset-contents/fleet.yaml

```diff
@@ -0,0 +1,49 @@
+- apiVersion: "v1"
+  kindsRegexp: "^namespaces$"
+  resourceNameRegexp: "^fleet-|^cluster-fleet-"
+- apiVersion: "v1"
+  kindsRegexp: "^secrets$"
+  namespaceRegexp: "^fleet-|^cluster-fleet-"
+  labelSelectors:
+    matchExpressions:
+      - key: "owner"
+        operator: "NotIn"
+        values: ["helm"]
+      - key: "fleet.cattle.io/managed"
+        operator: "In"
+        values: ["true"]
+- apiVersion: "v1"
+  kindsRegexp: "^serviceaccounts$"
+  namespaceRegexp: "^fleet-|^cluster-fleet-"
+- apiVersion: "v1"
+  kindsRegexp: "^configmaps$"
+  namespaceRegexp: "^fleet-|^cluster-fleet-"
+- apiVersion: "rbac.authorization.k8s.io/v1"
+  kindsRegexp: "^roles$|^rolebindings$"
+  namespaceRegexp: "^fleet-|^cluster-fleet-"
+- apiVersion: "rbac.authorization.k8s.io/v1"
+  kindsRegexp: "^clusterrolebindings$"
+  resourceNameRegexp: "^fleet-|^gitjob-"
```

Comment:
Gitjob was never tracked as a regexp:

```yaml
- apiVersion: "rbac.authorization.k8s.io/v1"
  kindsRegexp: "^clusterrolebindings$"
  resourceNameRegexp: "^cattle-|^clusterrolebinding-|^globaladmin-user-|^grb-u-|^fleet-"
  resourceNames:
    - "eks-operator"
    - "rancher-operator"
    - "gitjob-binding"
```

What other gitjob- bindings do we want to collect? is there a reason why we collect only gitob bindings but not the namespaces, secrets, serviceaccounts, configmaps, or rolebindings?

Reply:
It is only called "gitjob-binding". As for the second part, are you asking if gitjobs create those unique resources?

Reply:
Yes, but more broadly I guess my question is whether that would ever be in scope for the gitjob chart or if we are sure it will never be in scope.

If so, maybe we should future-proof this chart to include pick up all resources with `gitjob` as the name or `gitjob-*` by default?

Reply:
There's no harm in adding both, so I will.